app/javascript/dashboard/components/widgets/Avatar.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div
+    class="avatar-container"
+    :style="[style, customStyle]"
+    aria-hidden="true"
+  >
+    <span>{{ userInitial }}</span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Avatar',
+  props: {
+    username: {
+      type: String,
+    },
+    backgroundColor: {
+      type: String,
+    },
+    color: {
+      type: String,
+    },
+    customStyle: {
+      type: Object,
+    },
+    size: {
+      type: Number,
+      default: 40,
+    },
+    src: {
+      type: String,
+    },
+    rounded: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  computed: {
+    style() {
+      return {
+        display: 'flex',
+        width: `${this.size}px`,
+        height: `${this.size}px`,
+        borderRadius: this.rounded ? '50%' : 0,
+        lineHeight: `${this.size + Math.floor(this.size / 20)}px`,
+        fontWeight: 'bold',
+        alignItems: 'center',
+        justifyContent: 'center',
+        textAlign: 'center',
+        backgroundColor: this.backgroundColor,
+        font: `${Math.floor(this.size / 2.5)}px/${
+          this.size
+        }px Helvetica, Arial, sans-serif`,
```

Comment:
Could you please remove font family from here? I would like to have font-family inherited from body.

Reply:
Of course. Now the look of the avatar is coherent with the rest of the screen.
Done!